src/config.ts

```diff
@@ -23,7 +24,7 @@ export interface LoadGeneratorConfiguration {
 export const readConfig = (): ActionConfig => {
   const gatlingEnterpriseUrl = getGatlingEnterpriseUrlConfig();
   const config = { gatlingEnterpriseUrl, api: getApiConfig(gatlingEnterpriseUrl), run: getRunConfig() };
-  core.debug("Parsed configuration: " + JSON.stringify({ api: { ...config.api, apiToken: "*****" }, run: config.run }));
+  logDebug("Parsed configuration: " + JSON.stringify({ api: { ...config.api, apiToken: "*****" }, run: config.run }));
```

Comment:
unrelated to this change, but I think that the apiToken should be masked by GitHub itself if it's provided as a secret.

Reply:
Yes, secrets should be automatically redacted from the logs. The GitHub documentation still cautions users: "GitHub automatically redacts secrets printed to the log, but you should avoid printing secrets to the log intentionally". There can be limitations (or bugs) to this automated redaction system.

Reply:
As it debug log only, it is intentional, isn't it?

Reply:
Mmh... I don't think I understand what you mean? The point is that we should _not_ be doing it intentionally. Especially since we don't know the user's intent if they enable debug logging (which is not a particularly fine grained setting on GitHub Actions - they may have enabled debug logging to look at the debug logs from another step, or even another workflow entirely).